locales/en-us.json

```diff
@@ -176,7 +176,7 @@
   },
   "instructor": {
     "yourprofile": "Your Instructor Profile",
-    "yourprofiletagline": "Verified instructors can edit and contribute to the LibreTexts libraries.",
+    "yourprofiletagline": "Verified instructors can edit and contribute to the LibreTexts libraries. Either a Bio URL or Additional Information is required to submit your verification request.",
```

Comment:
This will need to be a new element/text entry, because the `yourprofiletagline` shows even when an instructor is already verified.